web/app.yaml

```diff
@@ -1,5 +1,4 @@
 runtime: python37
-default_expiration: "0d 3h"
```

Comment:
Instead of reverting to default, can we be explicit here that we want it to be 10 minutes?